web/src/click_handlers.js

```diff
@@ -388,6 +407,46 @@ export function initialize() {
         $row.find(".markdown_preview").show();
     });
 
+    $("body").on("mouseenter", ".compose_control_button", () => {
+        const buttons = document.querySelectorAll(`
+            .compose_control_button[data-tooltip-template-id],
+            .compose_control_button[data-tippy-content],
+            .compose_control_button_container
+        `);
+
+        const tippyInstances = [...buttons].map((button) => {
+            const tippyContent = button.dataset.tippyContent;
+            const tooltipTemplateId = button.dataset.tooltipTemplateId;
```

Comment:
There is a major issue with this logic, that being, we are attaching multiple Tippy instances over all the compose control buttons, over and over again when hovering above any one of them. In my manual testing, hovering multiple times would eventually lead to a stage where the tooltips become laggy and unresponsive.

Reply:
We should let the Tippyjs library handle the creation and deletion of the tooltips, and thus we should only call the `tippy()` constructor on the appropriate selectors once.

Reply:
https://atomiks.github.io/tippyjs/v6/addons/#singleton gives us the example of how we should go on with using the singletons addon.

Reply:
Updated the code so that tippy creation only triggers when the user mouseenters the complete```compose-control-buttons-container``` using tippy() constructor.